src/Nethermind/Nethermind.Merge.Plugin.Test/BaseEngineModuleTests.cs

```diff
@@ -161,6 +162,7 @@ protected IEngineRpcModule CreateEngineModule(MergeTestBlockchain chain, ISyncCo
             new ExchangeTransitionConfigurationV1Handler(chain.PoSSwitcher, chain.LogManager),
             new ExchangeCapabilitiesHandler(capabilitiesProvider, chain.LogManager),
             new GetBlobsHandler(chain.TxPool),
+            NullEngineRequestsTracker.Instance,
```

Comment:
Would a substitute be enough?

```suggestion
            Substitute.For<IEngineRequestsTracker>(),
```